conftest.py

```diff
@@ -29,3 +33,31 @@ def enable_console_logging():
 def csrf_check_client() -> Client:
     """A Django test client instance that enforces CSRF checks."""
     return Client(enforce_csrf_checks=True)
+
+
+@pytest.fixture()
+def root_page() -> Page:
+    """Returns the root page. Useful for adding pages in tests."""
+    return Page.objects.filter(depth=1).get()
+
+
+@pytest.fixture()
+def home_page() -> HomePage:
+    """Returns the home page."""
+    return HomePage.objects.first()
```

Comment:
Nit: Since there's only ever meant to be one homepage
```suggestion
    return HomePage.objects.get()
```

Reply:
But we won't, since we will have translations